### ISSUE TITLE
False positive when using class_alias function

### ISSUE THREAD

**Issue by kubawerlos** [maglnet/ComposerRequireChecker#108]
When using function `class_alias` there is false positive result for alias.

Example:
```
git clone git@github.com:PHP-CS-Fixer/phpunit-constraint-isidenticalstring.git
cd phpunit-constraint-isidenticalstring
composer u
composer u -d dev-tools
dev-tools/vendor/bin/composer-require-checker check
```

and the result will be:
```
The following unknown symbols were found:
+----------------------------------------------------------------------------+--------------------+
| unknown symbol                                                             | guessed dependency |
+----------------------------------------------------------------------------+--------------------+
| PhpCsFixer\PhpunitConstraintIsIdenticalString\Constraint\IsIdenticalString |                    |
+----------------------------------------------------------------------------+--------------------+
```

and it is only used as an alias in `class_alias` function.

**Comment by Ocramius** [comment]
That's because `class_alias()` is a dynamic expression, not a static symbol declaration: I don't think `class_alias()` is recognised at all
